src/api/endpoints/healthcheck_email/healthcheck_email.py

```diff
@@ -0,0 +1,55 @@
+"""Module to test Notifications being sent out."""
+# system imports
+from flask import request, current_app
+from flask_restplus import Resource
```

Comment:
Please use `flask_restful`, this package was removed from the application `flask_restplus`